packages/react-notion-x/src/components/sync-pointer-block.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  SyncPointerBlock as SyncPointerBlockType,
+  Block as BlockType
+} from 'notion-types'
+import React from 'react'
+
+import { NotionBlockRenderer } from '../renderer'
+
+export const SyncPointerBlock: React.FC<{
+  block: BlockType
+  level: number
+}> = (props) => {
+  let { block, level } = props
+
+  if (!block) {
+    if (process.env.NODE_ENV !== 'production') {
+      console.warn('missing block', block.id)
+    }
+    return null
+  }
+  const syncPointerBlock = block as SyncPointerBlockType
+  const referencePointerId =
+    syncPointerBlock?.format?.transclusion_reference_pointer.id
+  if (referencePointerId == null) return
+
+  if (referencePointerId == null) return null
+
+  return (
+    <NotionBlockRenderer
```

Comment:
i think `hideBlockId` won't get passed down properly in this case.. it should probably be part of the context that's setup when we initialize `NotionContextProvider`

Reply:
this is a minor thing though and i don't think it should block this PR

Reply:
And what's the hideBlockId do?

Reply:
@normdoow I added that in https://github.com/NotionX/react-notion-x/pull/87 - it removes the block ID from HTML classes

Reply:
ahh, got it.